etc/omero.properties

```diff
@@ -258,7 +258,7 @@ omero.scripts.cache.cron=0 0 0 * * ?
 # for how long.
 #
 # For more information, see
-# http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/cache/CacheBuilderSpec.html
+# https://github.com/google/guava/blob/master/guava/src/com/google/common/cache/CacheBuilderSpec.java
```

Comment:
The main issue here is that the Javadoc link becomes a source code link on the master branch. I would use the occasion to tie the link to the version of guava used in OMERO (currently 17.0 /cc @mtbc). We might also want to use the Javadoc published on GitHub pages i.e. http://google.github.io/guava/releases/17.0/api/docs/ although it is unclear to me on whether this should be the reference Javadoc URL for guava.